src/SourceGenerators/Uno.UI.SourceGenerators.Tests/Given_XBindParser.cs

```diff
@@ -37,9 +38,11 @@ bool IsStaticMethod(string name)
 
 				return false;
 			}
+
 			var props = XBindExpressionParser.ParseProperties(inputExpression, IsStaticMethod);
 
-			Assert.IsTrue(output.SequenceEqual(props), $"Expected [{string.Join(";", output)}], got [{string.Join(";", props)}]");
+			Assert.AreEqual(props.hasFunction, hasFunction, $"Expected hasFunction=true for [{inputExpression}]");
+			Assert.IsTrue(output.SequenceEqual(props.properties), $"Expected [{string.Join(";", output)}], got [{string.Join(";", props)}]");
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Refactor this method to add validation of parameter 'output' before using it.](https://app.codacy.com/gh/unoplatform/uno/pullRequest?prid=5087470)